src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -3,42 +3,13 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System;
-using EnsureThat;
-
 namespace Microsoft.Health.Dicom.Core.Features.CustomTag
 {
     /// <summary>
     /// External representation of a custom tag entry.
     /// </summary>
-    public class CustomTagEntry : IEquatable<CustomTagEntry>
+    public class CustomTagEntry
```

Comment:
This is all now `ExtendedQueryTag` in the other PR, right? So the net change is really just the addition of `PrivateCreator`?

Reply:
* There are 2 `CustomTagEntry`, one is server side (https://github.com/microsoft/dicom-server/blob/feature/customtags/src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs), the other is client side (https://github.com/microsoft/dicom-server/blob/feature/customtags/src/Microsoft.Health.Dicom.Client/Models/ExtendedQueryTag.cs).  There is user story https://microsofthealth.visualstudio.com/Health/_workitems/edit/80286 which Sneha is working on for renaming everything, so I didn't rename here.
*  Yes, the change is just addtional of `PrivateCreator`